creator/referral_tokens/mutations.py

```diff
@@ -82,14 +88,25 @@ def mutate(self, info, input):
 
         user = info.context.user
 
-        if not user.has_perm("referral_tokens.add_referraltoken"):
+        _, organization_id = from_global_id(input.get("organization"))
+        try:
+            organization = Organization.objects.get(pk=organization_id)
+        except Organization.DoesNotExist:
+            raise GraphQLError(f"Organizaton {organization_id} does not exist")
+
+        # User must be a member of the organization to send an invite
```

Comment:
It is. Currently only admins have the `add_referraltoken` permission. Since we are lacking the boundary between the organization admins vs study admins, I think this is a good enough check for now. We will revisit this when the boundary is better defined (RFC in progress).